### PR TITLE
Forms Automated Test - LPS-131229 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -69,12 +69,62 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```
Comment (CalumR23) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:88]: You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?

Comment (LucasAcacio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase]: Yes, I pushed the test with this change

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 2]
```diff
@@ -69,12 +69,62 @@ definition {
 	}
 
 	@description = "Verify that a Form can be submitted with the Paragraph Field"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeSubmitted {
 		property portal.acceptance = "false";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:74]: Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

Comment (LucasAcacio) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:75]: Yes, I just submitted this change along with the gotoAddField change

**Comment by CalumR23** [comment]
@LucasAcacio can you please remove the liferay-translations commit. We cannot review this PR while that commit is on the PR.

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 3]
```diff
@@ -69,12 +69,61 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		Form.editFieldLabelMetal(fieldValue = "Paragraph");
```
Comment (brittneyq) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:89]: Is this line needed for the test?

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
@diogocordeiro , @natocesarrego 

this should be ready for review!

**Comment by diogocordeiro** [comment]
Hi @LucasAcacio , there seems to be conflicting files with this PR for portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path
would you mind making the appropriate changes?

**Comment by brittneyq** [comment]
ci:test:sf

**Pull request closed without merging**